android/build.gradle

```diff
@@ -115,10 +93,23 @@ file("$REACT_NATIVE_DIR/ReactAndroid/gradle.properties").withInputStream { react
 def REACT_NATIVE_VERSION = reactProperties.getProperty("VERSION_NAME")
 def REACT_NATIVE_MINOR_VERSION = REACT_NATIVE_VERSION.startsWith("0.0.0-") ? 1000 : REACT_NATIVE_VERSION.split("\\.")[1].toInteger()
 
+
+def objects = objects
```

Comment:
why? BTW why this PR is different than https://github.com/software-mansion/react-native-reanimated/pull/4350/files?

Reply:
The RHS `objects` is actually a getter, and assigning its result to a variable makes it possible to capture in closure.

As for the difference between this and Reanimated, it does more complex checks, together with checking multiple locations. I can make it the same as in Reanimated.

Reply:
Thank you for the explanation! Yeah, let's move the implementation of this task into a class as in Reanimated (`abstract class NoMultipleInstancesAssertionTask extends DefaultTask`).